purchase_operating_unit/readme/ROADMAP.rst

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
remove line

Reply:
@elicoidal this why there are two blank lines in the generated readme, no magic :)

Reply:
True (I saw it later sorry) although I still think my question can be valid: why add a line after the title by default?
@sbidoul

Reply:
Sorry, my bad. Sometimes travis gives some errors with that.